gpbft/options.go

```diff
@@ -17,8 +17,9 @@ type options struct {
 	delta                time.Duration
 	deltaBackOffExponent float64
 	// tracer traces logic logs for debugging and simulation purposes.
-	tracer          Tracer
-	initialInstance uint64
+	tracer             Tracer
```

Comment:
Nit: keep tracer as the last item? Somehow group the algorithmically-relevant parameters from the coding artefacts.